docs/content/en/docs-v0.46.x/user-guide/managing-controlplane/configuration-reference.md

```diff
@@ -31,7 +31,7 @@ spec:
 
 | Field | Type | Description | Required |
 |-|-|-|-|
-| type | string | Which type of data store should be used. Can be one of the following values<br>`FIRESTORE`, `MYSQL`. | Yes |
+| type | string | Which type of data store should be used. Can be one of the following values<br>`FIRESTORE`, `MYSQL`, `FILEDB`. | Yes |
```

Comment:
Guys, this is an alpha feature so I don't really want to mention this directly on this page, wdyt?

Reply:
If you guys still want to mention it somewhere, I think we can add a new line to this page 
👉  https://pipecd.dev/docs-v0.46.x/feature-status/#control-plane

Be sure to mark that `Support file store as data store` feature as alpha feature 😉

Reply:
@t-kikuc please address this 👀

Reply:
@khanhtc1202 

> Be sure to mark that Support file store as data store feature as alpha feature 

That's good, i'll do so

Reply:
I did it